### ISSUE TITLE
Add Newsblur support

### ISSUE THREAD

**Issue by bil-elmoussaoui** [jangernert/FeedReader#28]
https://newsblur.com/api

**Comment by ptvyqmnq** [comment]
Could a bounty be added for this? I know Newsblur users would be interested in adding support.

**Comment by jangernert** [comment]
https://www.bountysource.com/issues/26359411-add-newsblur-support

**Comment by ptvyqmnq** [comment]
But there's no amount given...

**Comment by jangernert** [comment]
You mean an amount to aim for, at which I will implement this?

**Comment by ptvyqmnq** [comment]
@jangernert Yes, I am wary of giving money if there is not a reasonable chance of the feature being implemented as I am a user of Newsblur.

**Comment by jangernert** [comment]
An estimate for what? A time when it will be done? No!
But up to this point every back-end got implemented reasonably quick.
But it will definitely not land in 2.0 anymore. Will have to wait for 2.2

**Comment by ptvyqmnq** [comment]
No, not time, money. I don't expect features to be done for free if they are not a priority for you. But if like #35 you request an amount for your work, contributors may be willing to pay.

Either way, I respect your work and you are free to make your own decisions as it is your project.

**Comment by jangernert** [comment]
I was pretty sure I already did that. I set it to 150$ just like for ownCloud News. But I can't check right now because Bountysource is bugging out for me. Just displays a toolbar with a white page.

**Comment by ptvyqmnq** [comment]
>I set it to 150$ just like for ownCloud News.

I've never seen that when I checked. Thanks for clearing it up.
